packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -222,7 +224,12 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
           extracted,
         }),
         ...extraPostcssPlugins,
-        autoprefixer(),
+        postcssPresetEnv({
```

Comment:
It looks like `postcss-preset-env` also has a `browsers` option.  Can this be set as well?  This would ideally ensure that all the Browserslist usage is synchronized.